### PR TITLE
Create new bot version message types v1

### PR DESCRIPTION
**Pull request by VjeraTurk**
## Why this should be merged/ How this works

- Upgrade protocol and bot version to 9 which introduces message type   
`ActivityProductInfo`

- Replaced v1alpha with v1

- Followed defined order and grouped Request Types;

- Extend Partner Plugin with examples for `ActivityProductInfo` (and missing `ActivityProductList`)
______ 

Additional Fixes:

**Added `UnmarshalContent` for missing message types!
Added missing request types to `MessageCategory` check!
Fixed compression test**

## How it was tested

With API calls to partner plugin mockup from Postman using grpc reflection

### CODE REVIEW

File: .github/workflows/docker.yml
Comment (VjeraTurk) [inline .github/workflows/docker.yml:1]: these changes are coming from c4t (not merged/rebased to dev)

File: docker-compose.yml
Comment (SamJaarsma) [inline docker-compose.yml:1]: This list is not complete, I think it should be:

"GetNetworkFeeRequest,GetPartnerConfigurationRequest,PingRequest,AccommodationProductListRequest,AccommodationProductInfoRequest,AccommodationSearchRequest,ActivityProductListRequest,ActivityProductInfoRequest,ActivitySearchRequest,TransportSearchRequest,MintRequest,ValidationRequest,SeatMapRequest,SeatMapAvailabilityRequest,CountryEntryRequirementsRequest"

I also grouped them by type, so it easier to check
There are probably other aspects to check for the missing messages

File: internal/messaging/types.go [hunk 1]
```diff
@@ -21,37 +21,39 @@ import (
...
+	*networkv1.GetNetworkFeeRequest
+	*partnerv1.GetPartnerConfigurationRequest
+	*bookv1.MintRequest
+	*bookv1.ValidationRequest
+	*pingv1.PingRequest
+	*transportv1.TransportSearchRequest
+	*seat_mapv1.SeatMapRequest
+	*seat_mapv1.SeatMapAvailabilityRequest
+	*infov1.CountryEntryRequirementsRequest
+	*activityv1.ActivityProductInfoRequest
```
Comment (SamJaarsma) [inline internal/messaging/types.go]: here the list is complete... Group by type?

**Review by SamJaarsma — COMMENTED** [review]
In general it looks like a version change requires a lot of changes. Can this be parameterized?

File: docker-compose.yml
Comment (VjeraTurk) [reply docker-compose.yml:1]: @SamJaarsma 
I checked to see if each Request Type appeared in the code the same amount of time and have found a few places where Request Types were missing si I added it. I enforced grouping everywhere in the code and order in configuration.

Comment (SamJaarsma) [reply docker-compose.yml:1]: cool!

File: internal/messaging/types.go (hunk 1 above)
Comment (SamJaarsma) [reply internal/messaging/types.go]: great

**Review by mo-c4t — APPROVED** [review]

**Comment by VjeraTurk** [comment]
> In general it looks like a version change requires a lot of changes. Can this be parameterized?

@SamJaarsma As discussed today, this has been planned for the refactoring phase with the idea of auto-generating the code.
https://github.com/chain4travel/camino-private-issues/issues/549#issuecomment-2328379761

**Comment by SamJaarsma** [comment]
> > In general it looks like a version change requires a lot of changes. Can this be parameterized?
> 
> @SamJaarsma As discussed today, this has been planned for the refactoring phase with the idea of auto-generating the code. [chain4travel/camino-private-issues#549 (comment)](https://github.com/chain4travel/camino-private-issues/issues/549#issuecomment-2328379761)

perfect!

**Review by SamJaarsma — APPROVED** [review]
LGTM

**Pull request merged**